.github/actions/node/build/action.yml

```diff
@@ -0,0 +1,35 @@
+name: NodeJS build
+
+inputs:
+  NODE_VERSION:
+    required: true
+    type: string
+  RELEASE_ENVIRONMENT:
+    required: true
+    type: string
+  WORKING_DIRECTORY:
+    required: true
+    type: string
+
+runs:
+    using: composite
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js ${{ inputs.NODE_VERSION }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ inputs.NODE_VERSION }}
+          cache: 'npm'
+
+      - name: Install dependencies
+        shell: bash
```

Comment:
Rispetto agli altri step ho notato che qui specifici il tipo di shell. A cosa serve?

Reply:
Shell è obbligatorio quando creo una composite action con **run**

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsshell